### PR TITLE
Add action that opens (creating if necessary) aws credentials file

### PR DESCRIPTION
**Pull request by kiiadi**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Description

Allow customers to edit their credentials file inside the IDE, creating a template file if it the file does not already exist.

## Related Issue(s)

#163

## Testing

New tests added to cover this feature

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [x] Local run of `gradlew check` succeeds
- [x] My code follows the code style of this project
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [ ] A short description of the change has been added to the **CHANGELOG**

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/AwsSettingsPanel.kt [hunk 1]
```diff
@@ -103,24 +106,58 @@ private class AwsSettingsPanel(private val project: Project) : StatusBarWidget,
     }
 }
 
-private class ChangeAccountSettingsAction(private val accountSettingsManager: ProjectAccountSettingsManager) :
+class SettingsSelectorAction : AnAction(message("configure.toolkit")) {
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/AwsSettingsPanel.kt]: DumbAware

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialWriter.kt [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+        file.writeText(
+            """
+                [default]
+                aws_access_key_id=
+                aws_secret_access_key=
+            """.trimIndent()
+        )
+
+        file.setReadable(true, false)
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialWriter.kt]: Shouldn't this be true, true so only owner can read it?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialWriter.kt [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+import java.nio.file.Paths
+import java.util.regex.Pattern
+
+class CreateOrUpdateCredentialProfilesAction : AnAction(message("configure.toolkit.upsert_credentials.action")), DumbAware {
+    private val credentialCreator = CreateOrUpdateCredentialProfiles.getInstance()
+
+    override fun actionPerformed(e: AnActionEvent) {
+        val project = e.getRequiredData(DataKeys.PROJECT)
+        credentialCreator(project) {
+            Messages.showOkCancelDialog(
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialWriter.kt]: Messages has a back door to set an expected yes/no, if you want to simplify this.
`Messages.setTestDialog(TestDialog.OK)`

File: jetbrains-core/tst/software/aws/toolkits/jetbrains/core/credentials/DefaultCredentialFileWriterTest.kt [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        val baseFolder = folderRule.newFolder()
+        baseFolder.setExecutable(true, false)
+        baseFolder.setWritable(true, false)
+        baseFolder.setReadable(true, false)
+        val file = Paths.get(baseFolder.absolutePath, "credentials").toFile()
+
+        val sut = DefaultCredentialFileWriter
+        sut.createFile(file)
+
+        assertThat(Files.getPosixFilePermissions(file.toPath())).matches("rw-r--r--")
```
Comment (abrooksv) [inline jetbrains-core/tst/software/aws/toolkits/jetbrains/core/credentials/DefaultCredentialFileWriterTest.kt]: Should be 600

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialWriter.kt (hunk 3 above)
Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialWriter.kt]: Oooh - awesome - good call. Much simplified.

**Pull request merged**